Tools/Scripts/webkitpy/style/checker.py

```diff
@@ -459,7 +459,7 @@
     re.compile('.*TestExpectations$'),
     re.compile('.*TestExpectations.json$'),
     # Avoid imported WebDriverTests python machinery
-    re.compile('(?!WebDriverTests).{0,14}.*.py$'),
+    re.compile('^LayoutTests.(?!imported).*\\.py$'),
```

Comment:
There are a lot of python scripts into `WebDriverTests`. Are we sure we want to exclude them, instead of adding LayoutTests too.

Reply:
```
_NEVER_SKIPPED_FILES contained a regexp
"re.compile('(?!WebDriverTests).{0,14}.*.py$')" which matches all py
files under all directory but `WebDriverTests`.
```

Ah understood.